source/wp-content/themes/wporg-developer-2023/src/code-return-value/style.scss

```
@@ -2,5 +2,12 @@
 	.return-type {
 		font-family: var(--wp--preset--font-family--monospace);
 		font-size: var(--wp--preset--font-size--small);
+		color: var(--wp--preset--color--syntax-red);
+	}
+
+	.type {
+		font-family: var(--wp--preset--font-family--monospace);
+		font-size: var(--wp--preset--font-size--small);
+		color: var(--wp--preset--color--syntax-red);
```

Combine these? The rules are the same.
```suggestion
	.return-type,
	.type {
		font-family: var(--wp--preset--font-family--monospace);
		font-size: var(--wp--preset--font-size--small);
		color: var(--wp--preset--color--syntax-red);
```